.github/dependabot.yml

```
@@ -4,6 +4,8 @@
 # https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 
 version: 2
+env:
+  CHROMATIC_PROJECT_TOKEN : ${{ ${{ secrets.CHROMATIC_PROJECT_TOKEN }} }}
```

Remove extra wrapping operators
```suggestion
  CHROMATIC_PROJECT_TOKEN : ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

---

I'm looking through dependabot.yml docs, and I don't think `env` key works here. ctrl+F'ing through the docs shows that "secrets." only works in keys that are otherwise documented, and nothing like the above is shown as possible

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#configuration-options-for-private-registries